sites/partners/src/pages/users/index.tsx

```diff
@@ -153,15 +153,9 @@ const Users = () => {
       </Head>
       <PageHeader className={"relative md:pt-16"} title={t("nav.users")}>
         <div className="flex top-4 right-4 absolute z-50 flex-col items-center">
-          <SiteAlert type="success" timeout={5000} dismissable />
           {csvExportSuccess && (
-            <SiteAlert
-              timeout={5000}
-              dismissable
-              sticky={true}
-              alertMessage={{ message: t("users.exportSuccess"), type: "success" }}
```

Comment:
Curious if this points to a bug with the SiteAlert component itself or if it was more of an unusual case. If a bug, would it be appropriate to create a follow-up ticket in UI-C?

Reply:
I think we should move towards using the "setSiteAlertMessage" route instead of setting the alertMessage here. Seems like it is a more stable option. But I'm guessing there are use cases it doesn't work for. I'd also be curious what the plan for SiteAlert is in UI-Seeds